iio.h

```diff
@@ -59,10 +59,16 @@ typedef ptrdiff_t ssize_t;
 #define __pure __attribute__((pure))
 #endif
 #define __notused __attribute__((unused))
+# ifdef IIO_CHECK_RET
+#  define __check_ret __attribute__((warn_unused_result))
+# else
+#  define __check_ret
```

Comment:
after a bit of chatting with @dNechita we noticed the `# __2spaces__ define`
it's a good idea to avoid spaces there; since it can be a bit undefined behavior;
the precompilers typycally look for `#define`  keywords, and some precompilers are smart enough to ignore spaces, but some aren't

Reply:
Ok - changed.

It's just easier to read this way, and some are doing the same in the same file.